ledger/participant-integration-api/src/main/scala/platform/store/backend/oracle/OracleStorageBackend.scala

```diff
@@ -116,9 +116,10 @@ private[backend] object OracleStorageBackend
     override def arrayIntersectionNonEmptyClause(
         columnName: String,
         parties: Set[Ref.Party],
-    ): CompositeSql =
-      cSQL"EXISTS (SELECT 1 FROM JSON_TABLE(#$columnName, '$$[*]' columns (value PATH '$$')) WHERE value IN (${parties
-        .map(_.toString)}))"
+    ): CompositeSql = {
+      val inParties = parties.map(p => s"'$p'").mkString(", ")
+      cSQL"EXISTS (SELECT 1 FROM JSON_TABLE(#$columnName, '$$[*]' columns (value PATH '$$')) WHERE value IN (#$inParties))"
```

Comment:
Why this change? If I understand correctly, instead of using a query parameter, parties are now hardcoded in the SQL string.

Reply:
This is the fix for the issue with SQL query syntax error on Oracle. I still need to figure out the exact difference in rendering comparing to the proposed version. Without this change the active contract query lookup query fails on Oracle.

Reply:
If this value is really inlined then that potentially opens the query up to SQL injection attacks. Let's make sure that we pass the values as a parameter to the query.

Reply:
I'm on it!